unstructured/ingest/cli/cli.py

```diff
@@ -29,14 +29,15 @@ def ingest():
     cli_cmds.local,
     cli_cmds.elasticsearch,
     cli_cmds.confluence,
+    cli_cmds.sharepoint,
 ]
 
 for subcommand in subcommands:
-    ingest.add_command(subcommand())
+    ingest.add_command(subcommand())  # type: ignore
```

Comment:
why was that ignore line needed? I don't see this required or flagged in our linting in CI

Reply:
sure, but note that we don't see this issue in CI. meaning this is an issue with your local setup.

what does `mypy --version` return?
we should see [1.4.1](https://github.com/Unstructured-IO/unstructured/blob/f97300b7a0dc350bc1f21fd661fd90d7fae40684/requirements/test.txt#L55)